### DISCUSSION TITLE
ALOS-2 ScanSAR to stripmap interferometry

### DISCUSSION THREAD

**Discussion by fdelgadodelapuente** [isce-framework/isce2#836] [General]
Dear ISCE developers,

Is there a particular parameter to tune in the alos2App.py template for processing stripmap to ScanSAR interferograms?

Thanks,
Francisco

**Comment by CunrenLiang** [comment]
Francisco,

What issues do you have? Spatial resolutions of ScanSAR-stripmap interferometry are the same as those of ScanSAR. Ionosphere correction in low coherence areas might have some difficulties due to the limited range bandwidth of ScanSAR and the small coverage of stripmap.

Cunren

**Comment by fdelgadodelapuente** [comment]
Hi Cunren,

The issue was that my ScanSAR pair lacks one swath because it is full of water, but the code required the five swaths to be present. So I just copied one swath with the filename of the missing one, and that solved the problem.

Best,

**Reply by CunrenLiang** [reply]
Stripmap may overlap with two ScanSAR subswaths, so it requires all five subswaths when reading the data before it has the data to determine which two subswaths are needed. But this sounds wired from users' perspective. Glad to see users can always find a trick to get around an issue;-)
